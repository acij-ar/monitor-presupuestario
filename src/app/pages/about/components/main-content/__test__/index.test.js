const React = require('react');
const { shallow } = require('enzyme');

const AboutMainContent = require('..');

describe('AboutMainContent component', () => {
  it('should match snapshot', () => {
    const wrapper = shallow(<AboutMainContent/>);
    expect(wrapper).toMatchSnapshot();
  });
});
