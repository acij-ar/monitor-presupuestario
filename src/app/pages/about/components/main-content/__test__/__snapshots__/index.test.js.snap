// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AboutMainContent component should match snapshot 1`] = `
<ContentSection
  id="about-main-content"
  title="El Monitor Presupuestario"
>
  <p>
    Desde la Asociación Civil por la Igualdad y la Justicia (ACIJ), creamos el Monitor presupuestario con el objetivo de promover el acceso a información presupuestaria de manera sencilla, desagregada y accesible, y así contribuir a mejorar las posibilidades de incidencia e impacto en las acciones tendientes a visibilizar situaciones de discriminación, exclusión y garantía de derechos.
  </p>
  <p>
    El Monitor presupuestario es una plataforma que contiene información de los diferentes organismos, ministerios, secretarías, programas y actividades presupuestarias desde el año 2007, incluyendo el presupuesto proyectado por el Poder Ejecutivo para 2019.
  </p>
  <p>
    La plataforma permite monitorear la asignación presupuestaria de cada año, el presupuesto aprobado por el Congreso y el presupuesto vigente (con las modificaciones del Poder Ejecutivo), así como la ejecución de años anteriores.
  </p>
</ContentSection>
`;
